oss-cloud-backend/serverless.yml

```diff
@@ -59,6 +61,9 @@ functions:
       - http:
           path: updatePullRequests
           method: get
+      - schedule:
+          rate: rate(30 minutes)
```

Comment:
Maybe to make this configurable and set rate value as an env variable.

Reply:
Added environmental variable